sdk/data/azcosmos/cosmos_client.go

```diff
@@ -526,6 +529,14 @@ func (c *Client) accountEndpointUrl() *url.URL {
 	return c.endpointUrl
 }
 
+func (c *Client) addResponseValuesToSpan(ctx context.Context, resp *http.Response) {
+	span := c.internal.Tracer().SpanFromContext(ctx)
+	span.SetAttributes(
+		tracing.Attribute{Key: "db.cosmosdb.request_charge", Value: newResponse(resp).RequestCharge},
```

Comment:
I assume we're not worried about the implications of parsing this out of the response headers.

Reply:
It should be fine. [The code handles errors appropriately](https://github.com/kjg/azure-sdk-for-go/blob/5a97c4385564bc1f21c988a8aebf39dac25583e6/sdk/data/azcosmos/cosmos_response.go#L35) and will just return `0` if the header is invalid.

There's some minor duplication here, since most of the calling APIs are going to parse this as well, but I'm not sure it's worth reworking all the layering to prevent that duplication right now.